src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/CreateInvitationCommandHandler.cs

```diff
@@ -71,6 +79,8 @@ public async Task<Result<int>> Handle(CreateInvitationCommand request, Cancellat
             var meetingParticipants = new List<BuilderParticipant>();
             var mcPkgs = new List<McPkg>();
             var commPkgs = new List<CommPkg>();
+            
+            var project = await _projectRepository.GetProjectOnlyByNameAsync(request.ProjectName) ?? await AddProjectAsync(request, cancellationToken);
```

Comment:
suggestion: var project = await GetOrCreateProjectAsync(request, cancellationToken) 
GetOrCreateProjectAsync just return the oneliner you have here